charts/operator/crds/monitoring.googleapis.com_clusterpodmonitorings.yaml

```diff
@@ -695,23 +688,26 @@ spec:
                     maxItems: 100
                     type: array
                   metadata:
+                    default:
```

Comment:
question: is this the crux of the change? i.e moving the defaulting down a layer into the `.metadata` field?

Reply:
Ack thanks